python/servo/mutation/test.py

```diff
@@ -0,0 +1,35 @@
+import fileinput
+import re
+import subprocess
+import sys
+
+def mutate_line(file_name, line_number):
+    lines = open(file_name, 'r').readlines()
+    lines[line_number - 1] = re.sub(r'\s&&\s', ' || ', lines[line_number - 1])
+    out = open(file_name, 'w')
```

Comment:
This is opening the file twice for each mutation, which is fine for this small test, but might be problematic when you come to scale it up.